editors/vscode/src/client.ts

```diff
@@ -407,12 +407,28 @@ function getGoalsEnvContent(goals : Goal[]){
         codeHyps = `<div class="hypothesis">`;
         codeGoals = `<div class="pp_goals">`;
         
-        // check if this is a Type Goal
-    	// extra space is not a typo, this is defined in lsp_base.ml
-        if (goals[i].typeofgoal == "Typ "){
+        if (goals[i].typeofgoal == "Typ"){
```

Comment:
Hi @firewall2142 . Is there a way to factorize these two identical loops below? (Both types of goals now have a hyps field.)

Reply:
I made a PR for that : #527